lib/src/extensions.dart

```diff
@@ -67,6 +67,31 @@ extension DateTimeTimeExtension on DateTime {
     }
   }
 
+  /// [isEastAsianAgeReckoning] East Asian Age Reckoning
+  int calculateAge({bool isEastAsianAgeReckoning = false}) {
+    final currentDate = DateTime.now();
+
+    var age = currentDate.year - year;
+
+    if (isEastAsianAgeReckoning) {
+      return age + 1;
+    }
+
+    final month1 = currentDate.month;
+    final month2 = month;
+    if (month2 > month1) {
+      age--;
+    } else if (month1 == month2) {
+      final day1 = currentDate.day;
+      final day2 = day;
+      if (day2 > day1) {
+        age--;
+      }
+    }
+
+    return age;
```

Comment:
Hey, I might be wrong but wouldn't the snippet below be a rough estimation of this computation? 

```
DateTime.now().difference(this).inDays ~/ 365
```

Reply:
@jogboms 
I'm sorry. I can't understand what you want to say.
Can you fix it?

Reply:
I mean we can use the code snippet i shared as a replacement for the calculations you made and I highlighted.

Reply:
You mean. Is the code snippet replace to your `DateTime.now().difference(this).inDays ~/ 365`?
No, it isn't. Because yours doesn't think leap year.